### ISSUE TITLE
Posición de los subtítulos

### ISSUE THREAD

**Issue by neftalimich** [brai4u/FlowCast#3]
Disculpa, ¿hay manera de configurar la posición de los subtítulos? 
en mi televisión aparece demasiado arriba.
Gracias de antemano.

**Comment by brai4u** [comment]
No, por ahora no, pero estoy reescribiendo toda la app desde 0, y voy a agregar esa función 🤓
